.changes/unreleased/INTERNAL-20250123-114649.yaml

```diff
@@ -0,0 +1,6 @@
+kind: INTERNAL
```

Comment:
I would bypass the changelog policy for this commit by adding a `dependencies` label to the PR 🤫